faq.html

```diff
@@ -3,9 +3,128 @@
+                <li>30+ hours in the center</li>
+                <li>Minimum 2.2 GPA per semester</li>
+                <li>Progress Report</li>
+            </ul>
+        </ul>
+      
+        <ul>
+            <i><b>Yearly Requirements</i></b>
+            <ul>
+                <li>Apply for at least one scholarship, REU, or internship</li>
+                <li>Update your student education plan</li>
+            </ul>
+        </ul>
+
+    <p><b>Friend of MESA</b></p>
+        <ul>
+            <li>STEM major (calculus based)</li>
+            <li>Minimum 3 semesters remaining at COC</li>
+            <li>Minimum 2.0 GPA</li>
+        </ul>
+   
+        <ul>
+            <i><b>Semester Requirements</b></i>
+            <ul>
+                <li>Attend at least 1 activity/workshop</li>
+                <li>20+ hours in the center</li>
+                <li>Minimum 2.0 GPA per semester</li>
+            </ul>
+        </ul>
+
+    <h3>What Majors are Eligible to Apply?</h3>
+            <ul>
+                <li>Biology</li>
+                <li>Bio-Chemistry</li>
+                <li>Chemistry</li>
+                <li>Computer Science</li>
+                <li>Computer Engineering</li>
+                <li>Engineering</li>
+                <li>Geology</li>
+                <li>Math</li>
+                <li>Physics</li>
+                <li>Pre-Dental School</li>
+                <li>Pre-Medical School</li>
+                <li>Pre-Pharmacy School</li>
+                <li>Pre-Veterinary School</li>
+            </ul>
+
+    <h3>What Are the Benefits of Joining MESA?</h3>
+        <ul>
+            <li>Designated study space (MESA Center)</li>
+            <li>Support from tutors</li>
+            <li>Academic advisement</li>
+            <li>Priority registration</li>
+            <li>Leadership, scholarship, and research opportunities</li>
+            <li>Academic Excellence Workshops (AEW)</li>
+            <li>Guest speakers</li>
+            <li>UC application workshops</li>
+            <li>Field trips to UC and CSU campuses</li>
+            <li>Company tours to local industry</li>
+            <li>Locker Program</li>
+            <li>Textbook, laptop, and calculator rentals</li>
+        </ul>
+
+    <h2><u>MESA Center</u></h2>
+    <h3>What is the MESA Center?</h3>
+    <p>It is a dedicated space for studying and workshops, with areas for individual and group studying. The MESA Center has nine computers equipped with MatLab, MiniTab, and AutoCad. Students also have access to printers, textbooks, 3-D printers, and a kitchenette.</p>
+
+    <h3>Where is the MESA Center?</h3>
+    <p>It is located in Aliso Lab 114 at the Valencia campus.</p>
+
+    <h3>When is the MESA Center Open?</h3>
+    <p>[...] Semester Schedule
+        <ul>
+            Monday: 9:00am - 5:00pm<br>Tuesday: 9:00am - 5:00pm<br>Wednesday: 9:00am - 5:00pm<br>Thursday: 9:00am - 5:00pm<br>Friday: 9:00am - 12:00pm<br>
```

Comment:
Separate each day using the list item tag
```
<li>Monday: 9.00am - 5:00pm</li>
<li>Tuesday: 9:00am - 5:00pm</li>
```
This is more readable than having each day on one single line